CONTRIBUTING.md

```diff
@@ -27,7 +27,7 @@ Issues and pull requests contributing to this purpose have the best chance to ma
 ## Questions?
 
 Please DO NOT ask "how do I?" or usage questions via GitHub issues. Instead,
-use the [https://groups.google.com/forum/#!forum/nunjucks](mailing list).
+use the (mailing list)[https://groups.google.com/forum/#!forum/nunjucks].
```

Comment:
Shouldn't parentheses and brackets be the other way around? – Already in #1008, by the way :sweat_smile:

Reply:
I'll let the other one be merged in then.